### PR TITLE
[rac_kalasha] Added example text and id changed to include language name

### PR DESCRIPTION
**Pull request by LornaSIL**

### CODE REVIEW

**Comment by rachitralli** [comment]
@LornaSIL thanks

File: release/rac/rac_kalasha/source/rac_kalasha.kps [hunk 1]
```diff
@@ -106,7 +106,7 @@
       <ID>rac_kalasha</ID>
       <Version>1.0</Version>
```
Comment (darcywong00) [inline release/rac/rac_kalasha/source/rac_kalasha.kps]: Do you know if the keyboard/package version needs to be bumped. To force a rebuild of the package.

Comment (LornaSIL) [reply release/rac/rac_kalasha/source/rac_kalasha.kps]: You know better than I do. I was thinking meta data changes didn't need a version bump, but if it affects the build, then I guess it does.

**Comment by LornaSIL** [comment]
@darcywong00 Unfortunately, with a bump of the version I also felt that we should fix the language name, BCP codes, using the shared fonts, etc, etc. (so it's no longer just adding the "example" to the keyboard_info file).

**Comment by rachitralli** [comment]
@LornaSIL thanks for adding the example text.

**Pull request merged**